bodhi/server/models.py

```diff
@@ -691,6 +702,8 @@ class Build(Base):
     release_id = Column(Integer, ForeignKey('releases.id'))
     signed = Column(Boolean, default=False, nullable=False)
     update_id = Column(Integer, ForeignKey('updates.id'))
+    ci_status = Column(CiStatus.db_type(), default=None, nullable=True)
```

Comment:
It seems inconsistent for the migration to mark all updates as ignored, but for the default to be None.

I think I slightly prefer letting the old updates stay at None, but I think it's more important that we pick a consistent policy. Which do you prefer?